src/devices/Ws28xx/samples/Sk6812_Samples/Program.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device.Spi;
+using System.Drawing;
+using System.Threading;
+using Iot.Device.Graphics;
+using Iot.Device.Ws28xx;
+
+// Configure the count of pixels
+const int Count = 8;
+Console.Clear();
+
+SpiConnectionSettings settings = new(0, 0)
+{
+    ClockFrequency = 2_400_000,
```

Comment:
Please add comments in this sample to explain that this frequency is mandatory to reflect the comments in the code.

Reply:
if it's mandatory consider adding DefaultSpiClockFrequency public constant to the binding and using it in the sample

Reply:
> if it's mandatory consider adding DefaultSpiClockFrequency public constant to the binding and using it in the sample

Good point, done so.